RNTrueLayerPaymentsSDK/js/models/theme/Theme.ts

```diff
@@ -0,0 +1,10 @@
+import { AndroidColors } from "./AndroidColors";
+import { AndroidTypography } from "./AndroidTypography";
+
+export type Theme = {
+  android?: {
+    lightColors?: AndroidColors;
+    darkColors?: AndroidColors;
+    typography?: AndroidTypography;
+  }
+};
```

Comment:
[Suggestion]

```
export type AndroidTheme = Partial<{
  android: Partial<{
    lightColors: AndroidColors;
    darkColors: AndroidColors;
    typography: AndroidTypography;
  }>
}>;
```

Reply:
both name and type

Reply:
@enricocasadei  That;s the object ontaining both. in the next PR (the iOS one) it includes iOS stuff as well. So the name in fact is correct.